docs/source/whatsnew/v0.21.rst

```diff
@@ -114,6 +114,11 @@ Features
 
         plt.show()
 
+* Matthias Cuntz added a new convenience utility function
+  :func:`~cartopy.util.add_cyclic` for 2-dimensional coordinates lat and lon
+  when adding cyclic points to an array and optionally to the corresponding 1D
+  or 2D coordinate(s). (:pull:`1753`)
```

Comment:
Sorry, I don't quite understand what this is saying. The clause describing the utility function has several subclauses defining what it's useful for, but nothing really describing what it does.

Reply:
I expanded on the statement about add_cyclic_point in v0.11.rst, which said:

> * Andrew Dawson has added a new convenience utility function :func:`~cartopy.util.add_cyclic_point` to add a cyclic point to an array and optionally to a corresponding 1D coordinate. 

I am happy for any suggestions in better English ;-)

How's about:

> Matthias Cuntz added a new convenience utility function :func:`~cartopy.util.add_cyclic`. This is an extension of :func:`~cartopy.util.add_cyclic_point` for 2-dimensional coordinates lat and lon. (:pull:`1753`)

Reply:
Sorry forgot to ping: @QuLogic @greglucas

Reply:
That sounds good. No need to ping; we are subscribed automatically after commenting.